plugins/modules/gitlab_project_members.py

```diff
@@ -49,7 +49,7 @@
       - The access level for the user.
       - Required if O(state=present), user state is set to present.
     type: str
-    choices: ['guest', 'reporter', 'developer', 'maintainer']
+    choices: ['guest', 'reporter', 'developer', 'maintainer', 'owner']
```

Comment:
Please add a line to the description like:
```yaml
      - V(owner) was added in community.general 10.6.0.
```
Same applies to the other options below.